lib/src/components/mixins/collidable.dart

```diff
@@ -4,6 +4,7 @@ import '../../geometry/rectangle.dart';
 import 'hitbox.dart';
 
 mixin Collidable on Hitbox {
+  bool activeCollision = true;
```

Comment:
we should probably add a comment here explaining what this entails.

another thing that crossed my mind is that you split the list into two every tick. this allows you to dynamically change this boolean without issues. but could be even better for performance to keep the lists separated from the beginning. but just an idea.

Reply:
I think you are right. I have added some comments, but my English is not very good. I try my best to make the meaning clear and welcome to make corrections to them. As for the behavior of allocating and creating [list] in [collisiondetection], I think that although it may reduce some performance, its complexity can make the implementation as concise as possible



But I don't know how much the... Operator will affect performance

``` final allCollisionList = <Collidable>[

...activeCollisionList,

...noActiveCollisionList,

]; ```